docs/source/resources/faq.md

```diff
@@ -0,0 +1,129 @@
+
+Whole-query and CDN caches are most useful when an API receives many of the same queries. This commonly happens with public data, like content on pages of a site. Regardless of whether the API is used for public data or not, these caches almost always provide large performance benefits, and are highly recommended. You can read more about how to set up whole-query and CDN caching with `apollo-server` 2.0 [here](https://www.apollographql.com/docs/guides/performance.html).
+
+Partial query caching can be achieved by caching the responses from underlying services with something like Redis or Memcache. With this strategy, even if two queries look completely different from one another, if there is any duplication of data fetched, those results can be shared, preventing unnecessary traffic. The [`RESTDataSource`](https://www.apollographql.com/docs/apollo-server/features/data-sources.html) does this automatically if the appropriate `cache-control` headers are present in REST responses.
+
+#### How can I monitor the health of my GraphQL schema?
+
+Many apps and sites are powered almost completely by an API such as a GraphQL schema, so it's important to make sure the API is healthy at all times. Indicators of an unhealthy service include long response times, high resource usage, and unusual traffic patterns.
+
+[Apollo Engine](https://www.apollographql.com/engine) is a great tool to track many of these things. It allows close inspection of fields to make it easy to see both total response times as well as how long each field took to execute.
+
+Engine also has some integrations to make monitoring easier. The [Slack Integration](https://www.apollographql.com/docs/engine/integrations/slack.html#setup) delivers daily reports to give teams a quick overview of the health of their schema. The [DataDog integration](https://www.apollographql.com/docs/engine/integrations/datadog.html#Monitoring-with-Datadog) works with existing DataDog accounts, to help teams track schema performance. When things go wrong, Engine has [configurable alerts](https://www.apollographql.com/docs/engine/features/alerts.html) to notify teams of issues through PagerDuty or Slack.
+
+## Moving a product to GraphQL
+
+> You have a good understanding of how to write, deploy, and monitor GraphQL in production. You are looking to scale GraphQL features to your entire product line.
+
+#### How do I organize schema code to scale for a larger project?
+
+Keeping all schema code together makes sense for smaller projects, but once a project reaches a certain size, or has many people working on it, managing conflicts in the same file and code navigation can get difficult. Splitting types and resolvers up into smaller files can make this process much easier. Read [this blog post](https://blog.apollographql.com/modularizing-your-graphql-schema-code-d7f71d5ed5f2) to learn more.
+
+<!-- TODO: @jakedawkins Add server testing -->
```

Comment:
reminder to add this in

Reply:
This won't be ready in time, probably. I was thinking about adding this in a week or two when I can figure out the testing situation a bit. Do you think that's okay, @peggyrayzis?

Reply:
Yeah that's fine, please remember to add it because I will probs forget 😀